CODEOWNERS

```diff
@@ -1 +1 @@
-* @chasefleming @bjartek @bthaile
```

Comment:
Why is this file in this PR? This was from another commit.

Reply:
That is weird the file hasn't changed, it's the same as main. 
`* @bjartek @bthaile @ianthpun` is the same as main